src/Client.php

```diff
@@ -643,7 +644,7 @@ public function getSettlements(?string $startDate = null, ?string $endDate = nul
             'submerchant' => $subMerchant,
         ];
 
-        $query = http_build_query($parameters);
+        $query = http_build_query(array_filter($parameters));
```

Comment:
Running an array_filter without a callback function only iterates over the array entries here. Could you explain a bit of the logic you were going after?
Additionally it might be a bit misleading to touch settlements end point in "add payment report endpoint" pull request?

Reply:
Correction: array_filter is used for removal of empty parameters before sending the request